README.md

```diff
@@ -1,128 +1,207 @@
 ![Unconnected Marketeers](https://user-images.githubusercontent.com/23106652/174433923-0a6069e7-d10d-4107-8f0d-f4a8a9d56976.png)
-![grafik](https://user-images.githubusercontent.com/23106652/174433975-8f23b0b0-e48e-4be1-8cb7-d8e3e7ab6b8e.png)
 
-It replaces the original practice menu with this UI. It also adds a practice menu to "Start Game" in Great Fairy Wars and Uwabami Breakers, which don't have a built in practice mode. On certain spells you can select phases or change certain parameters.
+![Imperishable Night](https://user-images.githubusercontent.com/23106652/174433975-8f23b0b0-e48e-4be1-8cb7-d8e3e7ab6b8e.png)
+
+thprac replaces the original practice menu with a new UI shown in the image above; certain spells allow you to select phases or change certain parameters.
 
 ![Uwabami Breakers](https://user-images.githubusercontent.com/23106652/174434103-5fee7a13-0254-4602-a468-42330b985bb2.png)
 ![Great Fairy Wars](https://user-images.githubusercontent.com/23106652/174434121-063142f2-ef3d-4721-ab96-a252343cdb0e.png)
 
-The menu can be controlled with your keyboard or controller. With this method, you can select the option with UP or DOWN, and change the option with LEFT or RIGHT. You can also use the mouse. Press SHOT to confirm.
+This menu can be controlled using your keyboard or your controller. Using this, you can select options using the **UP** and **DOWN** keys and change the option using the **LEFT** and **RIGHT** keys and the **SHOT** key to select. You can also use your mouse to interact with the menu.
 
-Thprac will automatically save the options you input into this into any replay saved. When playing back the replay, thprac will automatically apply those options. **Replays saved when playing with thprac and the "Mode" set to "Custom" will not work with the unmodified game**
+thprac will also automatically save the options you selected into any replay saved. When playing back the replay, thprac will automatically apply said options. **Replays saved while thprac is active using "Custom" mode will NOT work with the unmodified game.**
 
-## Backspace menu
-In any supported game (except for TH09) you can press BACKSPACE to access the Backspace menu. It will show you options that can be activated with the F keys.
+## <a name="backspace-menu"></a> **Backspace Menu**
+In any supported game (**except Touhou 9**), you can press the **backspace** key to enter the backspace menu; the backspace menu will show you a variety of options that can be activated using the function (F) keys.
 
-NOTE for Steam Deck users: these options are also **clickable**. This means that you can use your left thumb to tap the options to toggle them. Therefore, you only need to map backspace in Steam Input.
+Do note that on the **Steam Deck** the options are clickable; you can use your left thumb to tap the options to toggle them. As a result, you only need to map the backspace key in Steam Input.
```

Comment:
These options are always clickable (you can click on them with the mouse when running on a desktop computer). I added that so that these options could be tapped on if you are using a Steam Deck. To say "that on the Steam Deck the options are clickable" is technically inaccurate though.